integration-tests/bats/log.bats

```diff
@@ -311,6 +311,34 @@ teardown() {
     [[ "$output" =~ $regex ]] || false
 }
 
+@test "log: --oneline removes all new lines" {
+    dolt commit --allow-empty -m "commit 1"
+    dolt commit --allow-empty -m "commit 2"
+    res=$(dolt log --oneline | wc -l)
+    [ "$res" -eq 3 ] # don't forget initial commit
```

Comment:
Probably should have some more checks on this output than just the number of lines